test/Conversion/TorchToTcp/data_movement.mlir

```diff
@@ -11,3 +11,17 @@ func.func @torch.aten.cat(%arg0: !torch.vtensor<[?,?],f32>, %arg1: !torch.vtenso
   %1 = torch.aten.cat %0, %int0 : !torch.list<vtensor>, !torch.int -> !torch.vtensor<[?,?],f32>
   return %1 : !torch.vtensor<[?,?],f32>
 }
+
+// CHECK-LABEL: @torch.aten.slice.Tensor
```

Comment:
nit: Specify `-split-input-file` and separate the tests with the `// -----` delimiter? Curious how it worked without that.